package.json

```diff
@@ -1,31 +1,27 @@
 {
   "name": "@ctt/crud-api",
-  "version": "1.6.0",
+  "version": "1.6.24",
   "description": "A microservice boilerplate",
   "main": "lib/index.js",
-  "types": "lib/index.d.ts",
   "scripts": {
-    "build": "yarn clean:build && tsc",
-    "postbuild": "prettier --write lib/**/**/*.*s",
+    "build": "tsc",
+    "build:pretty": "yarn build && yarn prettify:lib",
+    "prettify:lib": "prettier --write 'lib/**/**/**.*s'",
```

Comment:
Try mand maintain alphabetically order of scripts.

Reply:
Oh. Thanks 👍